includes/Vendor/SetupWizard.php

```diff
@@ -666,4 +653,83 @@ public function dokan_setup_ready() {
         </div>
         <?php
     }
+
+    /**
+     * Gets the URL for the next step in the wizard
+     *
+     * Handles special logic to skip the payment step if no withdrawal methods
+     * are active, preventing users from accessing an empty payment step
+     *
+     * @since 2.9.27
+     *
+     * @return string The URL for the next step
+     */
+    public function get_next_step_link(): string {
+        $keys = array_keys( $this->steps );
+        $step = array_search( $this->current_step, $keys, true );
+        ++$step;
+
+        // If next step is payment but there are no active methods, skip to the following step
+        if ( 'payment' === $keys[ $step ] && empty( dokan_withdraw_get_active_methods() ) ) {
```

Comment:
Pls handle the undefined index `'payment' === $keys[ $step ]`  with `'payment' === ($keys[ $step ] ?? '' )`.